tests/Unit/Core/Form/IdentifiableObject/CommandBuilder/Product/StockCommandsBuilderTest.php

```diff
@@ -155,7 +156,7 @@ public function getExpectedCommands(): Generator
         $command = $this->getSingleShopCommand();
         $localizedNotes = [
             '1' => 'hello',
-            '2', 'Goodbye',
```

Comment:
😮

Reply:
Yep, I was surprised by this change as well ^^'

Reply:
It as a typo present for a long time, we're supposed to provide a localized array So in the end it worked because it as interpreted as `['1' => "hello', 0 => '2', 3 => 'Goodbye']` (or something like that)